BuildScript/BuildTasks/TestConfigurationMetadata.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under 
+// the Apache License, Version 2.0 (the "License"); you may not use this 
+// file except in compliance with the License.  You may obtain a copy of the 
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT 
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the 
+// License for the specific language governing permissions and limitations
+// under the License.
+// 
+using System;
+
+namespace Remotion.BuildScript.BuildTasks
+{
+  public static class TestConfigurationMetadata
```

Comment:
In MetadataValueConstants there is a special warning about changing values, but here we don't need that. I think we can drop those remarks everywhere?

Reply:
I'm sorry, I can't follow. Which remarks are you referring to? I agree that we don't need special warnings about changing these, they are mainly there to avoid typos in stringly typed msbuild conversion code

Reply:
BuildScript/BuildTasks/MetadataValueConstants.cs line 23